openshift/configmap-template.yaml

```diff
@@ -50,6 +50,8 @@ objects:
       janusgraph-service-host: ${JANUSGRAPH_SERVICE_HOST}
       janusgraph-service-port: ${JANUSGRAPH_SERVICE_PORT}
       sentry-environment: ${SENTRY_ENVIRONMENT}
+      graph_service_host: "${GRAPH_SERVICE_HOST}"
+      graph_tls_path: "${GRAPH_TLS_PATH}"
```

Comment:
Now looking at it... we should be probably consistent with naming - dashes instead of underscores. Also, do we need to have GRAPH_TLS_PATH here? We could have it in each deployment, IMHO its not generic enough to have it here. WDYT?